packages/api/src/object/FetchPageArgs.ts

```diff
@@ -69,9 +70,11 @@ export interface AsyncIterArgs<
   R extends boolean = false,
   A extends Augments = {},
   S extends NullabilityAdherence = NullabilityAdherence.Default,
+  U extends boolean = false,
 > extends SelectArg<Q, K, R, S>, OrderByArg<Q, PropertyKeys<Q>> {
-  $augment?: A;
+  $augment?: U extends true ? never : A;
   $__EXPERIMENTAL_selectedObjectTypes?: string[];
+  $__EXPERIMENTAL_useUnstableInterfaces?: U;
```

Comment:
So I liked doing this better than exposing via experimental because I didn't like having to do `client(interfaceExperiemntSymbol)(myInterface).fetchPage...`, especially because then people will need to transition back to the normal syntax afterwards

Reply:
Only downside is that if you hit `$` this is easily exposed as an option, but idk we make it very clear its experimental and unstable